README.md

```diff
@@ -1 +1,62 @@
-# microverse-week-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Set up the project, the goal here is to master all of the tools and best practices about Linter and Gitflow.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Getting Started
```

Comment:
- [ ] In the README the section `Getting started` should explain how someone else can run or get the project in his/her local environment. you can include something like:
```
To get the content of this project locally you need to run this command in your terminal:
- `git clone yourlink
- cd projectName
```